src/hotspot/share/nmt/memReporter.cpp

```diff
@@ -249,7 +249,7 @@ void MemSummaryReporter::report_summary_of_type(MemTag mem_tag,
 
    // report malloc'd memory
   if (amount_in_current_scale(MAX2(malloc_memory->malloc_size(), pk_malloc)) > 0) {
-    print_malloc(malloc_memory->malloc_counter());
+    print_malloc(malloc_memory->malloc_counter(), mtNMT);
```

Comment:
Please revert, the original was correct. print_malloc is used for both summary and detail reports; for summary, we pass in mtNone deliberately since the report happens under a common tag.

Reply:
I need to look into this further, but I think "mtAllocated" would be better.